Makefile

```diff
@@ -14,6 +14,15 @@ install :
 	cd cli; \
 	make install
 
+vc:
+	cd virtcontainers; \
```

Comment:
This is fine, but it would be cleaner if all these rules (including the existing one on a separate commit) use the magic `-C` flag instead:

```
vc:
    make -C virtcontainers
```

Reply:
@jodh-intel Done.